Workbooks/SapMonitor2.0/OracleDb/OracleDb.workbook

```diff
@@ -266,7 +266,7 @@
                           "allowCustom": false
                         },
                         "value": {
-                          "durationMs": 43200000
+                          "durationMs": 86400000
```

Comment:
this changes the value selected by default. Are you sure this should be changed?

Reply:
discussed offline that it should be changed